src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -75,7 +77,7 @@ class EmojiPickerMenu extends Component {
         // prevent auto focus when open picker for mobile device
         this.shouldFocusInputOnScreenFocus = canFocusInputOnScreenFocus();
 
-        this.filterEmojis = _.debounce(this.filterEmojis.bind(this), 300);
+        this.filterEmojis = _.throttle(this.filterEmojis.bind(this), throttleTime);
```

Comment:
I used throttle because it invokes a function at specified intervals, so even if we type fast, it would still be called every 50/200 ms. With debounce, we would wait for 50/200 ms after the last user input before invoking the function. Hence, there were instances when users typed quickly and didn't see the immediate result.